gridData/CCP4.py

```diff
@@ -0,0 +1,211 @@
+# gridDataFormats --- python modules to read and write gridded data
+# Copyright (c) 2009-2014 Oliver Beckstein <orbeckst@gmail.com>
+# Released under the GNU Lesser General Public License, version 3 or later.
+
+"""
+:mod:`CCP4` --- the CCP4 volumetric data format
+===========================================
+
+.. _gOpenMol: http://www.ccp4.ac.uk/html/maplib.html#description
+
+The module provides a simple implementation of a reader for CCP4_
+*ccp4* files. CCP4 files are binary files. The :class:`CCP4` reader tries
+to guess the endianess of the file, but this can fail (with a
+:exc:`TypeError`); you are on your own in this case.
+
+Only the reader is implemented. If you want to write gridded data use a format
+that is more standard, such as OpenDX (see :mod:`OpenDX`).
+
+
+Background
+----------
+
+CCP4 format: http://www.ccp4.ac.uk/html/maplib.html#description
+
```

Comment:
No problem � just thought I ask.

Basically any format information for OpenDX and gOpenMol is also only available in caches � pretty scary. Nowadays I tend to WebCite pretty much any page that defines a format...

I also started copying & pasting format information into the docs, just so that at least I know what I used.

I don't think that CCP4 will go extinct any time soon but it is still inconvenient.

On 7 Jul, 2015, at 11:21, Jesse Johnson wrote:

> Unfortunately not, and Google Cache is the only other reliable source I know.